### ISSUE TITLE
Question: Does Spotify log songs played with spotifyd?

### ISSUE THREAD

**Issue by chilipepperhott** [Spotifyd/spotifyd#1064]
More specifically, I am curious if my listening time will be counted towards Spotify Wrapped at the end of the year.

If this is not the right place to ask this question, please feel no grief in removing the issue.

**Comment by eladyn** [comment]
HI! I unfortunately can't give you a direct answer, since I'm not sure wether it does count the listening time. However, I would tend to no, since the playback itself also is apparently not reported as you can see in the upstream issue https://github.com/librespot-org/librespot/issues/913, in [this discussion](https://github.com/librespot-org/librespot/discussions/626) and potentially also in [this one](https://github.com/librespot-org/librespot/discussions/947).

Maybe that helps you somehow with your question.

**Comment by BillyTomson** [comment]
Hmm, an interesting question! I also have a question. How can you become more popular in spotify? Does anyone have any advice on this?

**Comment by Cevin77** [comment]
You know, I just recently decided to become a singer of songs on the spotify platform! I released a very cool track and I wanted to raise it to the top of the songs as soon as possible. That's why I decided to [buy spotify plays](https://spotistar.com/buy-spotify-plays/), since raising the track to the top directly depends on it! I think that soon my song will be heard in many places and will become popular!

**Comment by Cevin77** [comment]
You know, I just recently decided to become a singer of songs on the spotify platform! I released a very cool track and I wanted to raise it to the top of the songs as soon as possible. That's why I decided to [buy spotify plays](https://spotistar.com/buy-spotify-plays/), since raising the track to the top directly depends on it! I think that soon my song will be heard in many places and will become popular!
